dev/ci/docker/README.md

```diff
@@ -4,10 +4,32 @@ This directory provides Docker images to be used by Coq's CI. The
 images do support Docker autobuild on `hub.docker.com` and Gitlab's
 private registry.
 
-Gitlab CI will build and tag a Docker by default for every job if the
-`SKIP_DOCKER` variable is not set to `false`. In Coq's CI, this
-variable is usually set to `false` indeed to avoid booting a useless
-job.
+If the `SKIP_DOCKER` Gitlab CI environment variable of a pipeline is not set to `true`,
+the CI will build a Docker image. This image will be
+stored in the [Gitlab container registry](https://gitlab.com/coq/coq/container_registry)
+under the name given by the `CACHEKEY` variable from
+the [Gitlab CI configuration file](../../../.gitlab-ci.yml).
+
+In Coq's default CI, `SKIP_DOCKER` is set so as to avoid running a lengthy redundant job.
+
+It can be used to regenerate a fresh Docker image on Gitlab through the following steps.
+- Change the `CACHEKEY` variable to a fresh name in the CI configuration in a new commit.
```

Comment:
Worth noting that the `CI configuration` file is `.gitlab-ci.yml` ?

Reply:
This is said above.

Reply:
Maybe adding "file described above"? I had noticed but in some cases CI configuration can refer to the CI configuation of the gitlab instance itself. As you have witnessed the process is quite confusing still :/